src/javascripts/components/visitor/displayVisitor/visitor.scss

```diff
@@ -0,0 +1,23 @@
+.card{
```

Comment:
This class selector will apply to *all* cards throughout the website because SCSS files are condensed to one style sheet.  If you only want this to apply to your cards, then you need to create your own unique class on the cards and then apply the style to that class instead of the bootstrap `.card` class. @nathanieltylerowens

Reply:
This goes for any of the generic bootstrap classes.  Always better to create your own class for specific styling that overrides the bootstrap defaults instead of modifying the defaults themselves.